src/main/java/array/Array.java

```diff
@@ -40,4 +40,11 @@
      * @param coordinates - position for deleting
      */
     void delete(C coordinates);
+
+    /**
+     * Visit the array (print for example)
+     *
+     * @param consumer - array's cell visitor
+     */
+    void visit(Consumer<T> consumer);
```

Comment:
There were no such method in here and signature of the method for visiting 1d and 2d array was different* SO I've got a suggestion to introduce common visit method. Please check the code below

Reply:
separate PR